spec/classes/server_spec.rb

```diff
@@ -31,7 +31,7 @@
             :servertype   => server_type,
             :storeconfigs => 'puppetdb',
             :environmentpath => '$confdir/environments',
-            :basemodulepath => '/etc/puppet/modules:/usr/share/puppet/modules',
+            :basemodulepath => ['/etc/puppet/modules', '/usr/share/puppet/modules'],
```

Comment:
we're already testing it, yes, https://github.com/puppetlabs-operations/puppet-puppet/blob/master/.travis.yml#L16 but i think it'd be also good to combine future and strict vars.